packages/web-react/src/types/collapse.ts

```diff
@@ -5,24 +5,24 @@ export type CollapseElementType = 'div' | 'article' | 'section' | 'main' | 'head
 
```

Comment:
So if we no longer use the `CollapseElementType`, we can easily delete it

Reply:
This is under consideration.

Reply:
We stay with this. Only allow modification of outer/root element. And I have also removed `span` because `div` as a block element is not allowed in `span`.